app/uk/gov/hmrc/teamsandrepositories/controller/TeamsRepositoriesController.scala

```diff
@@ -57,7 +57,7 @@ class TeamsRepositoriesController @Inject()(
   def repositoryDetails(name: String) = Action.async {
     val repoName = URLDecoder.decode(name, "UTF-8")
 
-    mongoTeamsAndReposPersister.getAllTeamsAndRepos(archived = None).map { allTeamsAndRepos =>
+    mongoTeamsAndReposPersister.getAllTeamsAndRepos(None).map { allTeamsAndRepos =>
       TeamRepositories.findRepositoryDetails(allTeamsAndRepos, repoName, urlTemplatesProvider.ciUrlTemplates) match {
```

Comment:
can you leave the named parameters, there is no context without an IDE
Similarly add to all calls to `getAllTeamsAndRepos` below